src/ledger/test/LedgerCloseMetaStreamTests.cpp

```diff
@@ -421,7 +421,8 @@ TEST_CASE("METADATA_DEBUG_LEDGERS works", "[metadebug]")
     while (lm.getLastClosedLedgerNum() < (2 * cfg.METADATA_DEBUG_LEDGERS))
     {
         clock.crank(false);
-        if (app->getWorkScheduler().allChildrenDone())
+        if (app->getWorkScheduler().allChildrenDone() &&
+            lm.getLastClosedLedgerNum() >= cfg.METADATA_DEBUG_LEDGERS)
```

Comment:
This is comparing an absolute ledger number to a count of ledgers -- is this intentional?